taskcluster/ci/lint/kind.yml

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+  - taskgraph.transforms.job:transforms
+  - taskgraph.transforms.task:transforms
+
+job-defaults:
+  worker-type: b-linux
+  worker:
+    docker-image: {in-tree: linux}
+    max-run-time: 1800
+
+jobs:
+  detekt:
+    description: 'detekt'
+    run:
+      using: run-task
+      cwd: '{checkout}'
```

Comment:
Nit: this can be bubble up in `job-defaults`

Reply:
Good call, I moved both `using` and `cwd` to `job-defaults`.